src/components/common/NavBar.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Nav, Navbar } from "react-bootstrap";
+import "./index.css";
+import { Link } from "react-router-dom";
+
+export const NavBar = () => {
+  return (
+    <div>
+      <Navbar className="navBar">
+        <Link to="/">
+          <Navbar.Brand>
+            <img
+              className="brandSpaces"
+              src="https://i.ibb.co/L9GcrMY/LLogo-2.png"
+              alt="Suljak Logo"
+            />
+          </Navbar.Brand>
+        </Link>
+
+        <Nav className="collapse navbar-collapse justify-content-end">
+          <Nav.Item className="navSpaces">
+            <Link to="/Contact" className="navLink">
+              CONTACT US
+            </Link>
+          </Nav.Item>
+
+          <Nav.Item className="navSpaces">
+            <Link to="/About" className="navLink">
+              ABOUT
```

Comment:
for capitalizing things, i actually prefer for this to say `About` (normal capitalization) and use css text-transform on navLink to make it capitalized. the reason si that it offers more flexibility in the future; it's easy to change between capitalize and non-capitalized in case you want to because all you have to do is change one css rule.